it/uk/gov/hmrc/incorporatedentityidentificationfrontend/api/controllers/JourneyControllerISpec.scala

```diff
@@ -234,6 +235,200 @@ class JourneyControllerISpec extends ComponentSpecHelper with JourneyStub with I
         )
       }
     }
+    "return captured data" when {
+      "the journeyId exists and verificationStatus is BusinessVerificationFail" in {
+        stubAuth(OK, successfulAuthResponse(Some(testInternalId)))
+        stubRetrieveIncorporatedEntityInformation(testJourneyId)(
+          status = OK,
+          body = Json.toJsObject(
+            IncorporatedEntityInformation(
+              companyProfile = testCompanyProfile,
+              optCtutr = Some(testCtutr),
+              identifiersMatch = false,
+              businessVerification = Some(BusinessVerificationFail),
+              registration = testSuccessfulRegistration
```

Comment:
If business verification fails the registration status will be not called.

Reply:
Richard again those tests help me to drive the json writer and reader the only meaning full data here is the BusinessVerification status ... the rest can be whatever....